charts/nodepool-labels-operator/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: nodepool-labels-operator
 description: Banzai K8s Nodepool Labels Operator
 apiVersion: v2
-appVersion: 0.1.2
-version: 0.2.6
+appVersion: 0.1.3
+version: 0.2.7
```

Comment:
**Question**: are the images/charts for those versions already available by the time this is merged?

Same for the `Values.image.tag`, also we should consider only using one of `Chart.appVersion` and `Values.image.tag` as those affect the same resources and might be confusing from a user perspective.
(We could comment the other one out, I would prefer keeping `Chart.appVersion` personally and use `Values.image.tag` for overrides only, not the defaults.)

Reply:
Thanks for noticing, tagging comes first so this is going to be in the next PR.